lib/fastlane/actions/make_changelog_from_jenkins.rb

```diff
@@ -0,0 +1,56 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      FL_CHANGELOG = :FL_CHANGELOG
+    end
+
+    class MakeChangelogFromJenkinsAction < Action
+      def self.run(params)
+        require 'nokogiri'
```

Comment:
We don't want `nokogiri` as a fastlane dependency. Could you solve this problem with some built-in libraries instead?